pom.xml

```diff
@@ -688,5 +691,14 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>debug</id>
+            <properties>
+                <connector.timeout>-1</connector.timeout>
```

Comment:
The timeout default value is already set in the Abstract* classes to `-1`, so you can remove this line